### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by DavidHeymann**
- Get token from spike
- Add redis to save token
- Request to kartofel with token

### CODE REVIEW

File: src/auth/auth.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    if (!verifyToken) {
+      try {
+        Auth.accessToken = (await Auth.axiosSpike.post(authParams.tokenPath, {
+          grant_type: "client_credentials",
+          audience: authParams.audience,
+          scope: authParams.scope.join(' '),
+        })).data.access_token;
+        await Auth.redis.set(Auth.keyName, Auth.accessToken);
+      } catch (error) {
+        throw Error(error.message);
```
Comment (weizmm) [inline src/auth/auth.js]: logs in winston

File: src/auth/auth.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      verifyToken = false;
+    }
+    if (!verifyToken) {
+      try {
+        Auth.accessToken = (await Auth.axiosSpike.post(authParams.tokenPath, {
+          grant_type: "client_credentials",
+          audience: authParams.audience,
+          scope: authParams.scope.join(' '),
+        })).data.access_token;
+        await Auth.redis.set(Auth.keyName, Auth.accessToken);
```
Comment (weizmm) [inline src/auth/auth.js]: winston logs

File: src/auth/auth.js (hunk 1 above)
Comment (DavidHeymann) [reply src/auth/auth.js]: Done

File: src/util/updateSpecificFields.js [hunk 3]
```diff
@@ -40,25 +40,12 @@ const updateSpecificFields = async (deepDiffArray, dataSource, person, akaRecord
     });
 
     try {
-        if (objForUpdate.directGroup) {
```
Comment (DavidHeymann) [inline src/util/updateSpecificFields.js]: ask matanel 🥚

File: src/schedule.js [hunk 4]
```diff
@@ -45,36 +62,22 @@ const devSchedual = async () => {
...
 
+    if(redis && redis.status === 'ready') redis.quit();
+});
 
-    //////////////////////MOCK-DELETE AT PRODACTION//////////////////////////////
-};
-devSchedual();
-///////////////////////////////////////////////////////////////////////////
-// });
+
```
Comment (DavidHeymann) [inline src/schedule.js]: Comment

**Pull request merged**